### ISSUE TITLE
Nested TypedArrays

### ISSUE THREAD

**Issue by dy** [jonschlinkert/arr-flatten#6]
Right now they are ignored.
Is that expected behavior?

```js
flat([[0,0,0,0], new Uint8Array([0,0,0,0])]) // [0,0,0,0]
```

**Comment by doowb** [comment]
> Is that expected behavior?

Yes, and I think that's correct. If there are different types of Typed Arrays, then what should the resulting array be? Will mixing types into a regular array cause any problems?
